components/edit_post_modal/edit_post_modal.tsx

```diff
@@ -179,6 +232,7 @@ class EditPostModal extends React.PureComponent {
 
         const {actions, editingPost} = this.props;
         const updatedPost = {
+            ...editingPost.post,
```

Comment:
I don't think we want to be adding extra fields when we patch the post. It could cause concurrency issues if we patch with a potentially out-of-date post object.

If this is needed to fix a type issue, I think we need to modify `editPost` to take a `Partial<Post>` instead of the full post object.